src/sdk/pynni/nni/platform/local.py

```diff
@@ -49,13 +49,16 @@ def request_next_parameter():
     })
     send_metric(metric)
 
-def get_parameters():
+def get_next_parameter():
     global _param_index
     params_file_name = ''
     if _multiphase and (_multiphase == 'true' or _multiphase == 'True'):
         params_file_name = ('parameter_{}.cfg'.format(_param_index), 'parameter.cfg')[_param_index == 0]
     else:
-        params_file_name = 'parameter.cfg'
+        if _param_index > 0:
```

Comment:
What will be the case if _param_index < 0?

Reply:
Thanks, assertion added.